code/modules/uplink/uplink_items.dm

```diff
@@ -500,19 +502,19 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A speed loader that contains seven additional lethal .357 Magnum rounds; compatible with the Nagant revolver. \
 			For when you really need a lot of things dead."
 	item = /obj/item/ammo_box/a357
-	cost = 2
+	cost = 1
```

Comment:
Wasnt this a high cost do to nukies getting it and using it for duel welding 120 harm with **tons** of ammo?